vtm/src/org/oscim/core/GeometryBuffer.java

```diff
@@ -178,8 +188,30 @@ public PointF getPoint(int i) {
         return out;
     }
 
+    /**
+     * Get the number of coordinates in point array.
+     * Equal to the next position to insert point coordinates in points array.
+     *
+     * @return the number of coordinates in point array.
+     */
+    public int getNumCoords() {
+        return numCoords;
+    }
+
```

Comment:
Would it be ok to add `getPointSize` in addition to `pointNextPos`?

Reply:
Mean the following?
```java
public int getPointsSize() {
    return pointNextPos;
}
```

Reply:
Exactly.

Reply:
Sure.